spec.html

```diff
@@ -44212,15 +44212,15 @@ <h1>
           1. If _isLittleEndian_ is *false*, reverse the order of the elements of _rawBytes_.
           1. If _type_ is ~float16~, then
             1. Let _value_ be the byte elements of _rawBytes_ concatenated and interpreted as a little-endian bit string encoding of an IEEE 754-2019 binary16 value.
-            1. If _value_ is an IEEE 754-2019 binary16 NaN value, return the *NaN* Number value.
+            1. If _value_ is an IEEE 754-2019 binary16 format NaN encoding, return *NaN*.
```

Comment:
This wording is pretty awkward - one doesn't normally speak of "an encoding" to refer to a particular bytestring ("an encoding" refers to the process or schema, not the result).

Maybe:
```suggestion
            1. If _value_ encodes a IEEE 754-2019 binary16 format NaN, return *NaN*.
```